content/javascript/concepts/strings/terms/slice/slice.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.slice()'
+Description: 'Returns a selected portion of a string.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Elements'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.slice()` method selects and returns a selected portion of a string.
+
+## Syntax
+
+```pseudo
+string.slice(start);
+
+string.slice(start, stop);
+```
+
+- The `start` index is optional and begins at zero (inclusive). Default is 0.
+- The `stop` index is also optional and defaults to the length of the string (not inclusive). Default is last element.
+
+## Example
+
```

Comment:
Like in the "Codebyte Example", let's include 1 or 2 sentences here about what is going on in the example snippet below.